compute/instances.go

```diff
@@ -252,6 +252,10 @@ type NetworkingInfo struct {
 	// Required
 	IPNetwork string `json:"ipnetwork,omitempty"`
 	// IP Network only.
+	// Set interface as default gateway for all traffic
+	// Optional
+	IsDefaultGateway bool `json:"is_default_gateway,omitempty"`
+	// IP Network only.
```

Comment:
Is the  `// IP Network only`comment  for `MACAddress` or for `IsDefaultGateway`. Just want to confirm that I'm not missing anything

Reply:
The diff tool is highlighting the wrong line. Following the pattern of the other comments - my addition was actually from line 254

```
	// IP Network only.
	// Set interface as default gateway for all traffic
	// Optional
	IsDefaultGateway bool `json:"is_default_gateway,omitempty"`
```

Line 258 applies to the next attribute.

Reply:
Ah I see!